File: Sort/selectionSort.go
package main

import "fmt"

func selectionSort(items []int) {
	n := len(items)
	for i := 0; i < n-1; i++ {
		position := i
		for j := i + 1; j < n; j++ {
			if items[j] < items[position] {
				position = j
			}
		}
		temp := items[i]
		items[i] = items[position]
		items[position] = temp

	}

}

func main() {
	items := []int{1, 2, 6, 8, 5, 4, 3, 7}
	fmt.Println(items)
	selectionSort(items)
	fmt.Println(items)
}

File: Sort/selectionSort.py
def selectionsort(a):
    n=len(a)
    for i in range(n-1):
        position=i 
        for j in range(i+1,n):
            if a[j]<a[position]: 
                position=j 
        temp=a[i] 
        a[i]=a[position]
        a[position]=temp


A = [3, 5, 8, 9, 6, 2]
print('Original Array:',A)
selectionsort(A)
print('Sorted Array:',A)
